app/src/androidTest/java/ch/epfl/polychef/fragments/UserProfileFragmentTest.java

```diff
@@ -104,6 +110,45 @@ public synchronized void recipesAreDisplayedWhenUserHasManyOfThem() throws Inter
         testUserRecipes(UserProfileFragment.nbOfRecipesLoadedAtATime + 1, 20);
     }
 
+    @Test
+    public synchronized void userAchievementsDisplayCorrectToast() throws InterruptedException {
+        mockUser.addRecipe("Recipe1");
+        mockUser.addRecipe("Recipe2");
```

Comment:
Should we keep this test?

Reply:
It is a test that tends to be flaky on the emulator. Here it does not achieve to click on the different pictures of User_profile (even with scrollTo). We should leave it commented for the moment, Mateo will try to fix it soon.